biothings/web/query/formatter.py

```diff
@@ -174,6 +185,12 @@ def transform(self, response, **options):
             template_miss = options.pop('template_miss', dict(found=False))
             responses = [self.transform(res, **options) for res in response]
             for tpl, res in zip(templates, responses):
+                total = res.get('total', {}).get('value') or 0
+                max_total += total
```

Comment:
why `max_total` is a sum of `total` values? Should be the maximum of an array of `total` values.

Reply:
hum, I think I misunderstood here.
I will update code to change max_total to the maximum of an array of total values.
@sengineer0

Reply:
It's updated! @sengineer0